docs/functions.rst

```diff
@@ -968,6 +968,7 @@ Outputs:
    `instanceID`, `string`, ID of the RDS instance
    `securityGroupID`, `[]string`, AWS Security Group IDs associated with the RDS instance
    `allocatedStorage`, `string`, Specifies the allocated storage size in gibibytes (GiB)
+   `dbSubnetGroup`, `string`, Specifies the DB Subnet group associated with the RDS instance
```

Comment:
this `dbSubnetGroup` will be associated with which RDS instance?

Reply:
RDS instance that we are trying to backup

Reply:
the function is to create an snapshot of RDS instance and not `create an RDS instance`